src/lambda_codebase/account_processing/process_account_files.py

```diff
@@ -129,19 +127,52 @@ def process_account(account_lookup, account):
 
 
 def process_account_list(all_accounts, accounts_in_file):
-    account_lookup = {
-        account["Name"]: account["Id"] for account in all_accounts
-    }
-    processed_accounts = list(map(
-        lambda account: process_account(
-            account_lookup=account_lookup,
-            account=account,
-        ),
-        accounts_in_file
-    ))
+    account_lookup = {account["Name"]: account["Id"] for account in all_accounts}
+    processed_accounts = list(
+        map(
+            lambda account: process_account(
+                account_lookup=account_lookup,
+                account=account,
+            ),
+            accounts_in_file,
+        )
+    )
     return processed_accounts
 
 
+def sanitize_account_name_for_snf(account_name):
+    unsupported_characters = [
+        "<",
+        ">",
+        "{",
+        "}",
+        "[",
+        "]",
+        " ",
+        "?",
+        "*",
+        '"',
+        "#",
+        "%",
+        "\\",
+        "^",
+        "|",
+        "~",
+        "`",
+        "$",
+        "&",
+        ",",
+        ";",
+        ":",
+        "/",
+    ]
+    sanitized_name = account_name[:30]
+    for char in unsupported_characters:
+        sanitized_name = sanitized_name.replace(char, "")
+
+    return sanitized_name
```

Comment:
I think we could change this to a regular expression that will take any non-alphanumeric or underscore char to an underscore. Like so:

```suggestion
def sanitize_account_name_for_snf(account_name):
    return re.sub('[^a-zA-Z0-9_]', '_', account_name[:30])
```

Reply:
Okay, makes sense. I'll make that change later in the day.